File: code/1036/1036.c
#include <math.h>
#include <stdio.h>
int main() {

  double A, B, C, r1, r2, delta;

  scanf("%lf %lf %lf", &A, &B, &C);

  delta = B * B - 4 * A * C;

  if (delta < 0 || A == 0) {
    printf("Impossivel calcular\n");
  } else {

    r1 = (-B + sqrt(delta)) / (2 * A);
    r2 = (-B - sqrt(delta)) / (2 * A);

    printf("R1 = %.5lf\nR2 = %.5lf\n", r1, r2);
  }
  return 0;
}

File: code/1036/1036.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const A = parseFloat(lines[0]);
const B = parseFloat(lines[1]);
const C = parseFloat(lines[2]);

const delta = B * B - 4 * A * C;

if (delta < 0 || A == 0) {
  console.log("Impossivel calcular");
} else {
  r1 = (-B + Math.sqrt(delta)) / (2 * A);
  r2 = (-B - Math.sqrt(delta)) / (2 * A);

  console.log(`R1 = ${r1.toFixed(5)}`);
  console.log(`R2 = ${r2.toFixed(5)}`);
}
